selenium/integration/starter.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This is a dummy test to make sure the webdriver works in the test env. Will be removed once we have real tests.